grails-app/views/cycleCount/_printCount.gsp

```diff
@@ -0,0 +1,130 @@
+<%@ page contentType="text/html;charset=UTF-8" defaultCodec="html" %>
```

Comment:
I'm surprised to see a GSP. I'm assuming this is how we create the PDF?

Reply:
Yeah, I didn't want to spend time figuring out how we could do it differently, than we used to, so I went with the existing approach